db/migrate/20210620144826_add_purpose_to_notifications.rb

```diff
@@ -0,0 +1,13 @@
+class AddPurposeToNotifications < ActiveRecord::Migration[5.2]
+  def up
+    add_column :notifications, :purpose, :string
+    Notification.find_each do |notification|
```

Comment:
All notifications currently in the wild should be from the covid medication reminders.

Reply:
We could be more robust by grabbing `Notification.where(subject: nil)` only. We can also set `Notification.where.not(subject: nil)` to `missed_visit_reminder` as well for thoroughness.